content/docs/glossary/liquidity-pool.mdx

```diff
@@ -61,12 +63,256 @@ There are a few possibilities with regard to authorization. The behavior of the
+
+</Alert>
+
+### Preamble
+
+For all of the following examples, we'll be working with three funded testnet accounts. If you'd like to follow along, generate some keypairs and fund them via the [friendbot](https://friendbot.stellar.org/?addr=YOUR_PUBLIC_KEY_HERE). If you use the secrets included in this guide, you may encounter problems with asset issuance or other account balance problems.
+
+The following code sets up the accounts and defines some helper functions. These should be familiar if you've played around with other examples, like [Clawbacks](./clawback.mdx#examples).
+
+<CodeExample>
+
+```js
+const sdk = require("stellar-sdk");
+
+let server = new sdk.Server("https://horizon-testnet.stellar.org");
+
+/// Helps simplify creating & signing a transaction.
+function buildTx(source, signer, ...ops) {
+  let tx = new sdk.TransactionBuilder(source, {
+    fee: sdk.BASE_FEE,
+    networkPassphrase: sdk.Networks.TESTNET,
+    withMuxing: true,
+  });
+  ops.forEach(op => tx.addOperation(op));
+  tx = tx.setTimeout(30).build();
+  tx.sign(signer);
+  return tx;
+}
+
+/// Returns the given asset pair in "protocol order."
+function orderAssets(A, B) {
+  return (sdk.Asset.compare(A, B) <= 0) ? [A, B] : [B, A];
+}
+
+/// Returns all of the accounts we'll be using.
+function getAccounts() {
+  return Promise.all(kps.map(kp => server.loadAccount(kp.publicKey())));
+}
+
+const kps = [
+  "SBGCD73TK2PTW2DQNWUYZSTCTHHVJPL4GZF3GVZMCDL6GYETYNAYOADN",
+  "SAAQFHI2FMSIC6OFPWZ3PDIIX3OF64RS3EB52VLYYZBX6GYB54TW3Q4U",
+  "SCJWYFTBDMDPAABHVJZE3DRMBRTEH4AIC5YUM54QGW57NUBM2XX6433P",
+].map(s => sdk.Keypair.fromSecret(s));
+
+// kp1 issues the assets
+const kp1 = kps[0];
+const [ A, B ] = orderAssets(...[
+  new sdk.Asset("A", kp1.publicKey()),
+  new sdk.Asset("B", kp1.publicKey()),
+]);
+
+/// Establishes trustlines and pays out to `recipientKp` for all `assets`.
+function distributeAssets(issuerKp, recipientKp, ...assets) {
+  return server
+    .loadAccount(issuerKp.publicKey())
+    .then(issuer => {
+      const ops = assets.map(asset => [
+        sdk.Operation.changeTrust({
+          source: recipientKp.publicKey(),
+          limit: "100000",
+          asset: asset,
+        }),
+        sdk.Operation.payment({
+          source: issuerKp.publicKey(),
+          destination: recipientKp.publicKey(),
+          amount: "100000",
+          asset: asset,
+        }),
+      ]).flat();
+
+      let tx = buildTx(issuer, issuerKp, ...ops);
+      tx.sign(recipientKp);
+      return server.submitTransaction(tx);
+    });
+}
+
+function preamble() {
+  return Promise.all([1, 2].map(i => distributeAssets(kp1, kps[i], A, B)));
+}
+```
+
+</CodeExample>
+
+Here, we use `distributeAssets()` to establish trustlines and set up initial balances of two custom assets (`A` and `B`, issued by `kp1`) for two accounts (`kp2` and `kp3`).
```

Comment:
It's worth highlighting that in order to add a trustline to a LP, an account must trust both LP assets. This was not obvious for some (SDF) developers that were playing around with LP, so the clearer we make it the better.